const Private = async (req, res) => {
  try {
  } catch (error) {
    console.error(error.message);
  }
};

export { Private };
